pages/dkp/kommander/2.2/install/mgmt-cluster-apps/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: layout.pug
+navigationTitle: Management cluster application requirements
+title: Management cluster application requirements
+menuWeight: 40
+excerpt: Management cluster application minimum resources and storage requirements
+beta: false
+enterprise: true
+---
+
+This section only details requirements for management cluster-specific applications. For the list of all platform applications, see [Platform Application Configuration Requirements](../../workspaces/applications/platform-applications/platform-application-requirements/).
+
+This table describes the workspace platform applications specific to the management cluster, minimum resource requirements, and minimum persistent storage requirements.
+
+| Name                          | Minimum Resources Suggested     | Minimum Persistent Storage Required | Deployed by Default |
+| ----------------------------- | ------------------------------- | ----------------------------------- | ------------------- |
+| centralized-grafana           |                                 |                                     |                     |
+| centralized-kubecostcert      |                                 |                                     | Yes                 |
+| chartmuseum                   |                                 |                                     | Yes                 |
```

Comment:
chartmuseum is only deployed by default for air-gapped installations, not sure how we want to surface that in this table

```suggestion
| chartmuseum                   |                                 | # of PVs: 1<br />PV sizes: 2Gi | Yes                 |
```